scripts/caput-pipeline

```diff
@@ -70,13 +70,32 @@ def queue(configfile, nosubmit=False):
         raise Exception('Job time is required.')
     clusterconf['time'] = conf['time']
 
+    if 'cluster' not in conf:
+        raise Exception('Cluster is required')
+    elif (conf['cluster'] == 'gpc'):
+        is_gpc = True
+        is_cedar = False
+    elif (conf['cluster'] == 'cedar'):
+        is_cedar = True
+        is_gpc = False
+    else:
+        raise Exception('Cluster name not recognized')
+
     # Set up optional PBS vars
     clusterconf['ompnum'] = conf['ompnum'] if 'ompnum' in conf else 8
     clusterconf['queue'] = conf['queue'] if 'queue' in conf else 'batch'
     clusterconf['pernode'] = conf['pernode'] if 'pernode' in conf else 1
-    clusterconf['ppn'] = conf['ppn'] if 'ppn' in conf else 8
     clusterconf['name'] = conf['name'] if 'name' in conf else 'job'
 
+    # Cedar-specific optional parameters
+    clusterconf['mem'] = conf['mem'] if 'mem' in conf else '128000M'
+    clusterconf['account'] = conf['account'] if 'account' in conf else 'rpp-krs'
+
+    if is_gpc:
```

Comment:
Actually, maybe a revised version of my previous comment is that there should be four options for `cluster`. `slurm` and `pbs` which just determine the submission template, but leave the default requirements (e.g. ppn and memory at the low values), and then `gpc` and `cedar` which set custom values for the `ppn` and `memory` defaults.

If you're feeling particularly energetic you could factor the last two out into a dictionary which hosts the default parameters. e.g.

```python
cluster_defaults = {
    'gpc' : {
        'ppn': 8, 'mem': '16000M', 'queue': 'pbs',
    },
    'cedar': {
        'ppn': 32, 'mem': '128000M', 'queue': 'slurm'
    }
}

# ...
# Lookup params
clusterconf['ppn'] = conf['ppn'] if 'ppn' in conf else cluster_defaults[cluster]['ppn']
```
then it's easy for us to extend this to new clusters